dbms/src/Flash/Coprocessor/ExecutionSummary.h

```diff
@@ -29,11 +30,15 @@ struct ExecutionSummary
     UInt64 num_iterations = 0;
     UInt64 concurrency = 0;
 
-    std::unique_ptr<DB::DM::ScanContext> scan_context = std::make_unique<DB::DM::ScanContext>();
+    DM::ScanContextPtr scan_context = std::make_shared<DB::DM::ScanContext>();
```

Comment:
due to c++ compile :(

Reply:
Don't get your point

Reply:
Because `unique_ptr` and `unordered_map` have a conflict, and it is difficult to solve in this case.